azdev/operations/help/__init__.py

```diff
@@ -105,6 +106,7 @@ def generate_cli_ref_docs(output_dir=None, output_type=None, all_profiles=None):
 
 
 def generate_extension_ref_docs(output_dir=None, output_type=None):
+    require_virtual_env()
```

Comment:
Is this an additional bug fix?

Reply:
This should be required in new codegen, I can move it to the codegen PR.